### PR TITLE
[stable/grafana] apply comment on last PR

### PR DESCRIPTION
**Pull request by obeyler**
#### What this PR does / why we need it:
apply good remark done on my last PR https://github.com/helm/charts/pull/15149 by @qrilka and @Kieran-Brown

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by qrilka** [comment]
@obeyler I have another comment on this: if you'll take a look at https://github.com/helm/charts/blob/master/stable/grafana/templates/configmap.yaml#L68 you'll see that `download_dashboards.sh` writes dashboard files into `/var/lib/grafana/dashboards/{{ $provider }}/{{ $key }}.json` and your fix creates only `/var/lib/grafana/dashboards/default` so using other provider name for a dashboard will end up write failure.

**Comment by obeyler** [comment]
> @obeyler I have another comment on this: if you'll take a look at https://github.com/helm/charts/blob/master/stable/grafana/templates/configmap.yaml#L68 you'll see that `download_dashboards.sh` writes dashboard files into `/var/lib/grafana/dashboards/{{ $provider }}/{{ $key }}.json` and your fix creates only `/var/lib/grafana/dashboards/default` so using other provider name for a dashboard will end up write failure.

In fact I see in the download-dashboards.sh script that it should create the directory of the provider https://github.com/helm/charts/blob/5c81c341dfb3d6deebc63fd8ec0bed0d524ec456/stable/grafana/templates/configmap.yaml#L49-L55a
It doesn't do that in case of "default"

**Comment by qrilka** [comment]
Oh, I missed that part, thanks, and it does if you add `dashboardProviders` entry, see e.g. https://github.com/helm/charts/issues/12352#issuecomment-476135441
So I think #15149 should be reverted and this connection between `dashboardProviders`/`dashboards` and actually `datasources` be better covered in documentation

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 3.5.11
+version: 3.5.12
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Please bump to 3.5.13. Thank you.

Comment (obeyler) [reply stable/grafana/Chart.yaml]: done

**Pull request closed without merging**